packages/bezier-react/.storybook/preview.js

```diff
@@ -105,45 +103,42 @@ function withFoundationProvider(Story, context) {
       : [lightBackgroundColor, darkBackgroundColor, lightThemeName, darkThemeName]
   })()
 
-  const foundation = {
-    ...Foundation,
-    ...customGlobalStyle,
-  }
-
-  const invertedFoundation = {
-    ...InvertedFoundation,
-    ...customGlobalStyle,
-  }
-
   return (
-    <div style={wrapperStyle}>
-      <BezierProvider 
-        foundation={foundation}
-        themeVarsScope=".theme"
-      >
-        <div className="theme" style={storyWrapperStyle}>
-          <div style={{ ...innerWrapperStyle, backgroundColor }}>
-            { Story(context) }
-          </div>
-          <Text bold color="bgtxt-absolute-black-light">
-            { themeName }
-          </Text>
-        </div>
-      </BezierProvider>
-      <BezierProvider
-        foundation={invertedFoundation}
-        themeVarsScope=".inverted-theme"
-      >
-        <div className="inverted-theme" style={storyWrapperStyle}>
-          <div style={{ ...innerWrapperStyle, backgroundColor: invertedBackgroundColor }}>
-            { Story(context) }
-          </div>
-          <Text bold color="bgtxt-absolute-black-light">
-            { invertedThemeName }
-          </Text>
+    <>
+      <GlobalStyle />
+
+      <BezierProvider foundation={foundation}>
+        <div style={wrapperStyle}>
```

Comment:
```suggestion
        <div style={wrapperStyle}>
```

`div` 상위의 `BezierProvider`는 빠져도 될 것 같은데 맞을까요?

Reply:
두개 테마만 사용되고 있으니 빠져도 괜찮아 보입니다 👀

Reply:
> root에 css variable을 추가하여 오버레이 색상이 잘 표시되도록 합니다.

`themeVarsScope` 를 제거해서 root에 CSS Variable을 생성하기 위해서 추가한 부분입니다.

Reply:
(구두 싱크) 이해했습니당